### DISCUSSION TITLE
Unity Catalog Support

### DISCUSSION THREAD

**Discussion by jaina15** [adidas/lakehouse-engine#6] [General]
Does this engine supports Databricks Unity Catalog?

**Comment by jmcorreia** [comment]
Hi @jaina15,

thank you for your question. 
Short answer, it should. If not already, we will do efforts to support it.
We are actually starting a test round of the engine in Unity enabled environments. In order for us to consider your concerns, would you be able to share more specifically which features/use cases you were thinking of?

Best Regards,
José Correia

**Reply by jaina15** [reply]
Hi @jmcorreia ,

Thank you for your response, as of now we are in the initial phase of analysis and as the requirements are that all target tables should be in UC (and can be both Managed as well as External). So, will we be able to create tables that supports 3 part naming convention (catalog_name.database_name.table_name)?

Regards,
Shubham Jain

**Reply by jmcorreia** [reply]
Hi @jaina15,
it should be supported, otherwise, please let us know, with all the details for us to replicate.
